### DISCUSSION TITLE
why I have been rejected for over six times?

### DISCUSSION THREAD

**Discussion by zhounan-sjtu** [community/community#52035] [GitHub Education]
I think I have edited my settings and information as verification problems and suggested solutions suggested.
But I was rejected again and again with such an absurd numb yellow face  -_-
Ok, rejection can be accepted if providing any reason?????   
So I can't stop wondering that HOW  A SIMPLE STUDENT SHOULD SUFFER THIS MOUNTAINOUS PREJUDICE ???????
Just give me the reason PLEASE ! ! !

**Comment by samus-aran** [comment]
Hi @sjtuBatnan 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐=

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
